packages/astro/src/core/build/static-build.ts

```diff
@@ -130,10 +136,27 @@ async function ssrBuild(opts: StaticBuildOptions, internals: BuildInternals, inp
 				input: [],
 				output: {
 					format: 'esm',
-					chunkFileNames: 'chunks/[name].[hash].mjs',
+					chunkFileNames: (chunkInfo) => {
+						if (chunkInfo.name.startsWith('pages/')) {
+							return `[name].[hash].mjs`;
```

Comment:
maybe in a pages/ subfolder so it's neater?

Reply:
Ah sorry this is not very clear, but yes that's what this does.